README.rst

```diff
@@ -60,9 +60,12 @@ And for full command line help, you can type::
 
 ``certbot-auto`` updates to the latest client release automatically.  And
 since ``certbot-auto`` is a wrapper to ``certbot``, it accepts exactly
-the same command line flags and arguments.  More details about this script and
-other installation methods can be found `in the User Guide
-<https://certbot.eff.org/docs/using.html#installation>`_.
+the same command line flags and arguments.  Throughout our and others' documentation
+you should substitute the name of the command that certbot.eff.org_ told you
+to use on your system. (``certbot-auto`` should always be run from the directory
+where it has been downloaded and invoked via ``./certbot-auto``).
```

Comment:
Should we also tell people they can copy it into their path, eg `/usr/local/bin/certbot-auto`?